libs/components/forms/src/lib/modules/checkbox/checkbox.component.scss

```diff
@@ -5,6 +5,11 @@
   display: inline-flex;
 }
 
+.sky-checkbox-container.sky-checkbox-stacked {
+  display: block;
```

Comment:
the default display of checkboxes is inline, and typically to stack them consumers put them in an unordered list. You can't apply margin to something displayed inline, but changing the display for all checkboxes would be a huge breaking change. I only change the display to block if the checkbox is being stacked.